backend/src/idl/policy.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package policy;
+
+option java_multiple_files = true;
+
+message PolicyDetails {
+    WorkSchedule policy = 1;
+    string administrator = 2;
+    string modified_at = 3;
+    string created_at = 4;
+    string last_scheduled = 5;
+}
+
+message WorkSchedule {
+    Schedule scheduling = 1;
+    repeated WorkDetails work = 2;
+    int32 vms_coverage = 3;
+    int32 vms_live = 4;
+    int32 profiles_discarded = 5;
+    int32 failures_observed = 6;
+    float effective_vmid_coverage_pct = 7;
+    string last_scheduled = 8;
+
+    message Schedule {
+        int32 duration = 1;
```

Comment:
Can duration be negative. Refer to datatypes and field names in recorder.proto, aggregations.proto